### ISSUE TITLE
Low accuracy on different datasets

### ISSUE THREAD

**Issue by GRIGORR** [Canjie-Luo/MORAN_v2#94]
Hi. Part of my training set contains space and more than 1 word. With another model I get about 95% on validation and about 90% on test. But using MORAN I get 86% on validation and 76% on test. I finetune your model by loading pretrained model except attention weights. Can the reason be the fact that in your case the model had seen only one word pictures and part of my data contains more than 1 words? 
Thanks.

**Comment by johnSmith1990** [comment]
Hi. By which model you get better result?

**Comment by Canjie-Luo** [comment]
Attention-based recognition network works well on short text (contains one word). For long text recognition, I recommend CRNN.
